dds_web/api/project.py

```diff
@@ -967,10 +967,16 @@ def get(self):
         project = dds_web.utils.collect_project(project_id=project_id)
         dds_web.utils.verify_project_access(project=project)
 
+        # if current user Researcher, show unit name instead of creator name
+        if auth.current_user().role not in ["Super Admin", "Unit Admin", "Unit Personnel"]:
+            project_creator = project.responsible_unit.name
+        else:
+            project_creator = project.creator.name
```

Comment:
This is just a personal preference that skips the `else`.
```suggestion
        project_creator = project.creator.name
        if auth.current_user().role not in ["Super Admin", "Unit Admin", "Unit Personnel"]:
            project_creator = project.responsible_unit.name
```